### PR TITLE
Adding boosted Z and inclusive DYJets cards for 2017 : DYJets_Zpt-150toInf_5f_LO_MLM and DYJets_HT_LO_MLM/DYJets_HT_mll50/DYJets_HT-incl (MadGraph5_aMCatNLO)

### PR DESCRIPTION
**Pull request by afanfani**
The parsing of the cards shows the following warnings:
WARNING in Proc Card: double 'define p'. Plese define the proton only one time!
WARNING in Proc/Run Card: plese use the correct pdf with b quark (nf==5)

There is indeed 'define p' twice, however since it was there in the pre2017 cards I left it.
For the PDF , is it ok to have  $DEFAULT_PDF_SETS = lhaid ? Or should it be 306000 = lhaid ?

### CODE REVIEW

**Comment by efeyazgan** [comment]
No. Please remove p and j definitions. 

For PDFs you should use
$DEFAULT_PDF_SETS = lhaid
$DEFAULT_PDF_MEMBERS  = reweight_PDF

**Comment by afanfani** [comment]
Duplicate p and j definition removed.

**Comment by efeyazgan** [comment]
Please remove the multiple lepton definitions as well. 

BTW we are waiting for the results of a final check of a parameter for MLM (as announced in https://hypernews.cern.ch/HyperNews/CMS/get/prep-ops/4427.html).

**Comment by afanfani** [comment]
Removed also multiple l definitions.
Thanks. I'll wait for the outcome of parametes for matching. At this stage I just wanted to test up to git pull request.

**Comment by afanfani** [comment]
Updated to " T        = pdfwgt "
commit 59c2257c0dadf58a4c1eb9b1e3c90041bc465a23

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJets_HT_LO_MLM/DYJets_HT_mll50/DYJets_HT-incl/DYJets_HT-incl_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.11                2013-06-21         *
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJets_HT_LO_MLM/DYJets_HT_mll50/DYJets_HT-incl/DYJets_HT-incl_proc_card.dat]: please remove the banner as certainly this is not anymore version 1.5.11 :-)

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJets_HT_LO_MLM/DYJets_HT_mll50/DYJets_HT-incl/DYJets_HT-incl_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+
+import model sm-ckm_no_b_mass
+define vl = ve vm vt
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJets_HT_LO_MLM/DYJets_HT_mll50/DYJets_HT-incl/DYJets_HT-incl_proc_card.dat]: remove the neutrinos as they are not used

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJets_HT_LO_MLM/DYJets_HT_mll50/DYJets_HT-incl/DYJets_HT-incl_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+define vl~ = ve~ vm~ vt~
+# Define multiparticle labels
+define p u c s d b u~ c~ s~ d~ b~ g
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJets_HT_LO_MLM/DYJets_HT_mll50/DYJets_HT-incl/DYJets_HT-incl_proc_card.dat]: we strongly recommend to remove all the definitions of protons and jets

**Comment by perrozzi** [comment]
@afanfani even if the PR is merged it would be good if you could implement the suggestions and make another PR, thanks a lot

**Pull request merged**